kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,24 @@
+package com.kodilla.exception.test;
+
+import java.awt.font.GlyphJustificationInfo;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class FlightFinder {
+
+        public void findFlight (Flight flight) throws RouteNotFoundException {
+
+            Map<String, Boolean> flights = new HashMap<>();
+            flights.put("Warsaw", true);
+            flights.put("Berlin", true);
+            flights.put("Frankfurt", true);
+            flights.put("Frankfurt", true);
+            flights.put("Gdańsk", true);
```

Comment:
Dlaczego wszystkie maja true? Niech niektore maja false. I to wtedy trzeba nizej obsluzyc.